lbrynet/core/Wallet.py

```diff
@@ -1051,7 +1069,8 @@ def handle_checks(results):
                 else:
                     log.warning("Something went wrong checking a balance. Peer: %s, account: %s,"
                                 "expected balance: %s, expected time: %s, count: %s, error: %s",
-                                str(balance[0]), str(balance[1]), str(balance[2]), str(balance[3]),
+                                str(balance[0]), str(balance[1]), str(
+                                    balance[2]), str(balance[3]),
                                 str(balance[4]), str(result.getErrorMessage()))
```

Comment:
These should not be wrapped in `str()` since the string formatting does this already.